tools/build_defs/pkg/rpm.bzl

```diff
@@ -87,26 +118,37 @@ pkg_rpm = rule(
             executable=True,
             allow_files=True),
     },
-    outputs = {
-        "out": "%{name}.rpm",
-        "rpm": "%{name}-%{architecture}.rpm",
-    },
+    outputs = _pkg_rpm_outputs,
     executable = False)
 """Creates an RPM format package from the data files.
 
 This runs rpmbuild (and requires it to be installed beforehand) to generate
 an RPM package based on the spec_file and data attributes.
 
+Two outputs are guaranteed to be produced: (1) "%{name}.rpm", and
```

Comment:
"Two outputs are guaranteed to be produced: "%{name}.rpm", and "%{name}-%{architecture}.rpm". If the "version" and "release" arguments are non-empty, a third output will be produced, following the RPM-recommended N-V-R.A format (Name-Version-Release.Architecture.rpm), "